app/renderer/css/preference.css

```diff
@@ -42,14 +42,6 @@ table {
   margin-bottom: 18px;
 }
 
-table tr:nth-child(even) {
-  background-color: rgba(250, 250, 250, 1);
-}
-
-table tr:nth-child(odd) {
-  background-color: rgba(255, 255, 255, 1);
-}
-
```

Comment:
Do you understand why this CSS resulted in non-alternating colors?  It's kinda puzzling.

Reply:
it's because of hidden elements in shortcuts section
for ex:
```
<tr ${process.platform === "darwin" ? "hidden" : ""}>
```